addon/components/kinetic-form.js

```diff
@@ -157,27 +127,20 @@ export default Component.extend({
     set(this, '_updatedFields', A());
   },
 
+
+
   actions: {
     updateProperty(key, value, validate = true) {
       if (get(this, 'readOnly')) return;
+
+      // START HACK
       set(this, `changeset.${key}`, value);
-      // HACK: ember-changeset will not set a property that is not valid. This is causing some bogus ui state so we need to manually set the property
```

Comment:
@cmackenz can we maintain the `//HACK: ... ` as editors are aware of this syntax and will highlight appropriately

Reply:
sure